CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+### Changelog
```

Comment:
Is this changelog for HED 3 and forward? If so it's quite confusing to have logs for HED v7.0.4 and v.7.0.5.

Reply:
I suggest that we label the versions for the HED restructuring as vreX.X.X rather than vX.X.X to distinguish.  I think I can go in and modify the git annotation tags.  Yes the version should be vre1.0.1.